api/apps/curations/index.coffee

```diff
@@ -0,0 +1,11 @@
+express = require 'express'
+routes = require './routes'
+{ setUser, authenticated, adminOnly } = require '../users/routes'
+
+app = module.exports = express()
+
+app.get '/curations', routes.index
+app.get '/curations/:id', routes.find, routes.show
+app.post '/curations', setUser, authenticated, adminOnly, routes.save
```

Comment:
actually this specific route doesn't get used and this can be removed, the UI doesn't allow creation of new curations, just editing existing ones.